target-platforms/packages/portal-base/src/build.js

```diff
@@ -184,3 +178,27 @@ function runSass() {
 		}
 	});
 }
+
+function runTsc() {
+	print(info`Running {tsc} compiler...`);
+
+	spawn('npx', ['tsc']);
```

Comment:
It is also possible that we have to split `portal-base` in two, because the configuration we impose on opinionated projects (those intentended to be deployed in a specific liferay version, following our opinionated build) may conflict with those of uniopinionated projects. So let's wait to see what happens before taking this decision, I would say...